src/main/java/com/microfocus/application/automation/tools/results/parser/antjunit/AntJUnitReportParserImpl.java

```diff
@@ -51,9 +52,9 @@
 		try {
 			return parseTestSetsFromAntJUnitReport(reportInputStream, testingFramework, testingTool);
 		} catch (Exception e) {
-		
-			e.printStackTrace();
-			throw new ReportParseException();
+			throw new ReportParseException(e);
+		} finally {
+			IOUtils.closeQuietly(reportInputStream);
```

Comment:
As of Java 7, closeQuietly is deprecated and even if not, we shouldn't swallow exceptions that could occur in that part, instead you could use try-with-resources and implement java.lang.AutoCloseable if not just use any try-catch inside the finally.